nilearn/regions/parcellations.py

```diff
@@ -153,6 +174,35 @@ class Parcellations(MultiPCA):
         This parameter is passed to image.resample_img. Please see the
         related documentation for details.
 
+    mask_strategy: {'background', 'epi' or 'template'}, optional
+        The strategy used to compute the mask: use 'background' if your
+        images present a clear homogeneous background, 'epi' if they
+        are raw EPI images, or you could use 'template' which will
+        extract the gray matter part of your data by resampling the MNI152
+        brain mask for your data's field of view.
+        Depending on this value, the mask will be computed from
+        masking.compute_background_mask, masking.compute_epi_mask or
+        masking.compute_gray_matter_mask. Default is 'epi'.
+
+    mask_args: dict, optional
+        If mask is None, these are additional parameters passed to
+        masking.compute_background_mask or masking.compute_epi_mask
+        to fine-tune mask computation. Please see the related documentation
+        for details.
+
+    scaling : bool, optional (default False)
+        Used only when the method selected is 'rena'. If scaling is True, each
+        cluster is scaled by the square root of its size, preserving the
+        l2-norm of the image.
+
+    n_iter : int, optional (default 10)
+        Used only when the method selected is 'rena'. Number of iterations of
+        the recursive neighbor agglomeration.
+
+    threshold : float in the open interval (0., 1.), optional (default 1e-7)
+        Used only when the method selected is 'rena'.
+        Threshold used to handle eccentricities.
```

Comment:
Also: should the ReNA threshold be exposed to the users in the Parcellation object? It seems like the default value should be good most of the time, and that for the rare case for which is it not, advanced users can use the ReNA object.

Reply:
Why make it complicated? We can add the advice in the parameter description that default is good most of the time yet keep it simple to experiment.

Reply:
@GaelVaroquaux 
What do you suggest please, I am not sure to understand... To me, the threshold is simply a precision parameter, that has no impact as soon as it is lower than something like 1e-4.
I am not sure if it is relevant to add a warning or something else since it is very unlikely that this parameter has any impact...